saleor/graphql/discount/schema.py

```diff
@@ -2,14 +2,12 @@
 from graphql_jwt.decorators import permission_required
 
 from ..core.fields import PrefetchingConnectionField
-# FIXME: Types are imported before mutations on purpose. Otherwise these types
```

Comment:
I don't have any errors regarding this, so I sorted this normally. Schema was created normally too as far as I can see.

Reply:
This was some really strange issue related to initialization of Graphene types. Maybe somehow we accidentally fixed it ;)